index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div id="main">
```

Comment:
- [ ] [REQUIRED] Please remove [`I think`](https://github.com/Agetuni/CodeReview_Microverse/pull/1/files#r913106727) because since you are suggesting it with this comment, you should say that you need to change it in a friendly way.

Reply:
[OPTIONAL] it would be better to use the semantic tag "main" here.